colcon-test/.lcovrc

```diff
@@ -0,0 +1,163 @@
+#
```

Comment:
The original file is here. https://gitlab.com/autowarefoundation/autoware.auto/AutowareAuto/-/blob/master/.lcovrc
lcovrc's default file is the same as .Auto, which makes it easier to compare coverage results between .Auto and Universe. For this reason, I have adopted this file.

Reply:
Why do we have to compare results with .Auto? :thinking: 
I think we don't have to care about it and that we should try to re-consider the best settings.

Reply:
> Why do we have to compare results with .Auto? thinking

This may be the case when comparing the percentage of tests written in .Auto and Universe.

>  I think we don't have to care about it and that we should try to re-consider the best settings.

I agree that we should consider the best configuration. However, I can't find anything wrong with the .Auto setting. For the use cases I envision, this setting is sufficient. 
If there is a setting you would like to change, please let me know.

Reply:
> This may be the case when comparing the percentage of tests written in .Auto and Universe.

Mm, what do we want to accomplish by that?

> If there is a setting you would like to change, please let me know.

To do that, could you share and explain what you have searched, please? :pray: